src/table/utils/color-utils.ts

```diff
@@ -24,7 +27,7 @@ export function resolveToRGBAorRGB(input) {
   // argb
   matches = /^argb\(\s*(\d{1,3})\s*,\s*(\d{1,3})\s*,\s*(\d{1,3})\s*,\s*(\d{1,3})\s*\)$/i.exec(input);
   if (matches) {
-    const a = Math.round(matches[1] / 2.55) / 100;
+    const a = Math.round(Number(matches[1]) / 2.55) / 100;
```

Comment:
rarely use Number(). How about '+' instead.

Reply:
ah yes that is better